main.py

```diff
@@ -83,5 +85,13 @@ def printQuantitativeBowTie(quantitativeBowTie):
     return 1
 
 
+def generate_trainingset_fault_tree(nr_events):
```

Comment:
Inconsistent naming scheme.

Reply:
Resolved everywhere else, naming scheme is now consistent. :D